packages/client/hmi-client/src/components/workflow/ops/model-config/tera-model-config.vue

```diff
@@ -154,26 +154,16 @@
 							</DataTable>
 						</AccordionTab>
 					</template>
-					<AccordionTab>
-						<template #header>
-							Parameters<span class="artifact-amount">({{ numParameters }})</span>
-						</template>
-						<tera-parameter-table-v2
-							v-if="!isEmpty(knobs.transientModelConfig) && !isEmpty(mmt.parameters)"
-							:model-configuration="knobs.transientModelConfig"
-							:mmt="mmt"
-							:mmt-params="mmtParams"
-							@update-parameter="
-								setParameterDistribution(knobs.transientModelConfig, $event.id, $event.distribution)
-							"
-							@update-source="
-								setParameterSource(knobs.transientModelConfig, $event.id, $event.value)
-							"
-						/>
-						<section v-else>
-							<p class="empty-section">No parameters found.</p>
-						</section>
-					</AccordionTab>
+				</Accordion>
+				<tera-parameter-table-v2
+					v-if="!isEmpty(knobs.transientModelConfig) && !isEmpty(mmt.parameters)"
+					:model-configuration="knobs.transientModelConfig"
+					:mmt="mmt"
+					:mmt-params="mmtParams"
+					@update-parameters="setParameterDistributions(knobs.transientModelConfig, $event)"
```

Comment:
can now update 1 or more parameters at a time

Reply:
Can we select only the available constant